exchange/exchange.go

```diff
@@ -123,16 +157,57 @@ func (e *exchange) buildBidResponse(liveAdapters []openrtb_ext.BidderName, adapt
 			// Possible performance improvement by grabbing a pointer to the current seatBid element, passing it to
 			// MakeSeatBid, and then building the seatBid in place, rather than copying. Probably more confusing than
 			// its worth
-			sb := e.makeSeatBid(adapterBids[a], a, adapterExtra)
+			sb := e.makeSeatBid(adapterBids[a], a, adapterExtra, targData)
 			seatBids = append(seatBids, *sb)
 		}
 	}
+	var err1 error = nil
+	if targData.targetFlag {
+		for id, bid := range targData.bid {
+			bidExt := new(openrtb_ext.ExtBid)
+			err1 = json.Unmarshal(bid.Ext, bidExt)
+			if err1 == nil {
```

Comment:
This error handling looks suspicious.

If one bid returns a malformed `Ext`, and another bid doesn't, shouldn't that be captured somewhere? It looks like `err1` gets discarded.

I see you capture `err = err1` below... but that's _outside_ the loop, so it would only note the final bid to be processed.